tutorials/cloudbuild-angular-universal/index.md

```diff
@@ -127,41 +130,41 @@ You will create a repository called `tour-of-heroes-universal`
         cp tmpfile package.json
         rm tmpfile
 
-8.  Add the package.json changes to Git.
+8.  Add the `package.json` changes to Git:
 
         git add package.json
 
-9.  Commit your changes to the Git repo
+9.  Commit your changes to the Git repository:
 
         git commit -m "pregenerate changes"
 
 ### Configure a Cloud Storage bucket and Cloud load balancer to host your Angular application on Cloud CDN
```

Comment:
Cloud Load Balancing

Reply:
Because this is a reference to an individual load balancer rather than to the service Cloud Load Balancing, I don't think that this mention should be changed to 'Cloud Load Balancing'. I'll remove the 'Cloud', though.